stock_archive_constraint/readme/USAGE.md

```diff
@@ -0,0 +1,3 @@
+1.  Go to Settings \> Users \> Edit a user and check the "Manage
+    Multiple Stock Locations" permission
+2.  Go to Inventory \> Settings \> Locations and disable one
```

Comment:
We should add :

3- Go to inventory > Products > Productos and try to archive one product associated to stock picking

I thing we should replace "disable one" for "archive" because that is the action the user should do to test this module

Reply:
Thank you for your review and comments. I have applied your suggestions. Please update your review. Thanks!